test/android_test.dart

```diff
@@ -149,5 +169,6 @@ String getAndroidManifestExample(String iconLine) {
     </application>
 
 </manifest>
-  '''.trim();
+  '''
+      .trim();
```

Comment:
All of the changes in `android_test.dart` should be from `dartfmt`.

Reply:
Thanks for the note!